### DISCUSSION TITLE
Looking for training and consultancy to migrate code from AS3

### DISCUSSION THREAD

**Discussion by DannyKodicek** [pixijs/pixijs#9075] [Jobs]
Hi, I'm a long-time Flash and Director developer and in need of some help in migrating our large AS3 library to Pixi. I'm looking for someone to help me get over my lack of understanding on certain core concepts and help me design a framework to make this process work as smoothly as possible.

I'm extremely experienced and knowledgeable about many things but woefully ignorant in others and need someone who can really clearly help me past the gaps in my knowledge. I'm new to both Pixi and Typescript, and indeed to Node.js. I imagine I'm looking for about 6 hours remote work spread over 2-3 days (UK time), but it's hard to be sure at the moment. 

If you can help me, or know someone who can, please get in touch with some details of your experience and availability. Thanks.

**Comment by miltoncandelero** [comment]
If you a looking for a freelancer you can also try the [PixiJS discord](https://discord.gg/QrnxmQUPGV) 💪 

If you are looking for a company you can ping where I work over at info@killabunnies.com.ar 

Full disclosure: I work for killabunnies and on my free time I help building PixiJS. Killabunnies and PixiJS are **not** related 😅

**Reply by DannyKodicek** [reply]
Hi, Milton, thanks very much! I did try Discord but didn't have any luck as yet. But we've sent you an email at Killabunnies.

**Comment by Article19** [comment]
Hey Danny - I remember your name from back in the Director days. I've been down the road you're on...having converted our entire game library from Director to Flash (AS2, then AS3) then to HTML5 (first with CreateJS, then PIXIJS). We don't use TypeScript or Node - but happy to help on other fronts.

You can reach me at article19.com
